src/app/cpr/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ if(NOT DRISHTI_BUILD_MIN_SIZE)
     target_compile_definitions(train_cpr PUBLIC DRISHTI_USE_IMSHOW)
   endif()
   
-  set_property(TARGET train_cpr PROPERTY FOLDER "app/console")
+  set_target_properties(train_cpr PROPERTIES FOLDER "app/console" INTERPROCEDURAL_OPTIMIZATION DRISHTI_EXE_IPO)
```

Comment:
Fixed.  Thanks.  I will need to setup gcc 5.0 locally to investigate that toolchain failure.